### PR TITLE
Deep branch new

### PR DESCRIPTION
**Pull request by sauradeep93**
Updates in packages and code

### CODE REVIEW

**Review by XiaoqZhang — COMMENTED** [review]
Did anyone test if this works?

File: environment_idigest.yml [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    - urllib3==2.0.2
+    - validators==0.20.0
+    - wcwidth==0.2.6
+    - webcolors==1.13
+    - webencodings==0.5.1
+    - websocket-client==1.5.2
+    - widgetsnbextension==3.6.4
+    - yarl==1.9.2
+    - zipp==3.15.0
+prefix: /Users/sauradeepmajumdar/miniconda3/envs/idigest
```
Comment (XiaoqZhang) [inline environment_idigest.yml:186]: This line can be removed

Comment (sauradeep93) [reply environment_idigest.yml]: have removed it .....

**Comment by sauradeep93** [comment]
> 



> 
it is working for me, but it would be good if someone else also runs it and installs the dependencies from the environment_idigest.yml file so as to make sure that all our versions are consistent

**Review by XiaoqZhang — APPROVED** [review]

**Pull request merged**